src/core/annotation.js

```diff
@@ -1088,6 +1088,8 @@ class ButtonWidgetAnnotation extends WidgetAnnotation {
         break;
       }
     }
+    this.checkedAppearance = normalAppearanceState.get(this.data.buttonValue);
```

Comment:
Let's add a newline above this line like we did for the checkboxes to separate the appearance from the button value.